_common

```diff
@@ -116,7 +116,8 @@ handle_unknown() {
         group_mailto=$(echo "$group_description" | perl -n -wE'm/.*MAILTO: (\S+).*/ and say $1' | head -1)
         group_mailto=${group_mailto:-$notification_address}
         if [[ -n "$group_mailto" ]]; then
-            echo "$header"$'\n'"Reason: $reason"$'\n'"$excerpt" | mailx -s "Unknown issue to be reviewed (Group $group_id)" -r "openqa-label-known-issues <$from_email>" "$group_mailto"
+            group_description=$(echo "$group_data" | runjq -r '.[0].name' ) || true
+            echo "$header"$'\n'"Reason: $reason"$'\n'"$excerpt" | mailx -s "Unknown issue to be reviewed (Group $group_id $group_name)" -r "openqa-label-known-issues <$from_email>" "$group_mailto"
```

Comment:
Where will `group_name` come from?

Reply:
From the line that was not meant to duplicate the existing variable 😅